Revert part of commit a53ae5000c439608b9dcf884b63d62b219e8fcc0.

It makes Singular segfault at least when calling pOne() and currRing is NULL.
diff -druN a/latest/Singular/iparith.cc b/latest/Singular/iparith.cc
--- a/latest/Singular/iparith.cc	2014-11-19 05:06:05.000000000 -0800
+++ b/latest/Singular/iparith.cc	2014-11-20 08:30:25.068389635 -0800
@@ -8477,7 +8477,6 @@
     sArithBase.sCmds[i].name);
     sArithBase.sCmds[i].alias=1;
   }
-  #if 0
   if (currRingHdl==NULL)
   {
     #ifdef SIQ
@@ -8493,7 +8492,6 @@
     }
     #endif
   }
-  #endif
   if (!expected_parms)
   {
     switch (tok)
